File: CodeWars/1.AreYouPlayingBanjo/Javascript/AreYouPlayingBanjo.js
const prompt = require('prompt-sync')();

function main() {
    var s = prompt("Enter your name : ");

    var result = PlaysBanjo(s);

    console.log(result);
}

function PlaysBanjo(name) {
    if (name[0].toLowerCase() == 'r') {
        return name + " plays banjo";
    } else {
        return name + " does not play banjo";
    }
}

main();

File: CodeWars/8Kyu/1.AreYouPlayingBanjo/Java/AreYouPlayingBanjo.java
import java.util.Scanner;

public class AreYouPlayingBanjo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter your name : ");
        String s = scanner.nextLine();

        System.out.println(PlaysBanjo(s));

        scanner.close();
    }

    public static String PlaysBanjo(String name) {
        char firstChar = name.charAt(0);
        char LowerCaseCharacter = Character.toLowerCase(firstChar);

        if (LowerCaseCharacter == 'r') {
            return name + " plays banjo";
        } else {
            return name + " does not play banjo";
        }
    }
}